Tab Options

abhinav162 opened the pull request:
Now user can input number of tabs to open

stym-rj commented:
- [ ] Try checking the PR once again, it's only opening 1 tab on any number of specified tabs.
- [ ] TODO: If user doesn't specifies the `No of tabs` to be opened, then set the default `No of tabs` to be opened to be **30** .

stym-rj commented:
It seems like you haven't pulled new code before committing your code.
Try pulling new code and update your code in that using `git pull origin master`

abhinav162 commented:
> * [x]  Try checking the PR once again, it's only opening 1 tab on any number of specified tabs.
> * [x]  TODO: If user doesn't specifies the `No of tabs` to be opened, then set the default `No of tabs` to be opened to be **30** .

Updated the code and fixed above issues
